include/flatcc/reflection/flatbuffers_common_reader.h

```diff
@@ -14,7 +14,7 @@
 #define __flatbuffers_read_vt(ID, offset, t)\
```

Comment:
I'd rather have changes to generated code run as a separate commit. Did you makes these changes manually, or did you run the script in
https://github.com/dvidelabs/flatcc/tree/master/reflection
?

Reply:
reverted